package/yast2.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2
-Version:        4.2.53
+Version:        4.2.54
 Release:        0
 Summary:        YaST2 Main Package
 License:        GPL-2.0-only
```

Comment:
as change is backward incompatble for packager you need conflict here with older packager. Otherwise `zypper up yast2` can result in crash as old packager willbe used.